MTL-1681 Add a notice for interface name changes (#1356)
Cray-HPE/docs-csm: install/prepare_configuration_payload.md

      extra command line options. It will expect to find all of the above files in the current working
      directory.

      > **`NOTE`**: For fresh installs and reinstalls of 1.2.X+ it is recommended that the `install-ncn-bond-members` option be double-checked. Systems 
      > with Mellanox PCIe cards will have new interface names.
      > 1. Compare the output of `lid` with the background document here for PCIe devices. Cross-reference the Vendor ID with the IDs in the table in [Vendor and Bus ID Identification](../background/ncn_networking.md#vendor-and-bus-id-identification).
      >
      >     ```bash
      >     <lan> <vid>:<did>
      >     pit# lid
      >     p238p1 8086:1521
      >     p238p2 8086:1521
      >     p238p3 15B3:1013
      >     p238p4 15B3:1013

>       >

Reviewer: It looks like you are missing the backticks here.
```suggestion
      >     ```
      >
```

>       > 1. Update `system_config.yaml`'s `install-ncn-bond-members` valye with the interface names cross-referenced from the `lid` output.

Reviewer 2: small typo

>       > 1. Update `system_config.yaml`'s `install-ncn-bond-members` valye with the interface names cross-referenced from the `lid` output.

Reviewer: ```suggestion
      > * Update `system_config.yaml`'s `install-ncn-bond-members` value with the interface names cross-referenced from the `lid` output.
```

>       > 1. _Alternatively_ run `csi config init --install-ncn-bond-members` with the discovered interface names.

Reviewer: ```suggestion
      > * _Alternatively_, run `csi config init --install-ncn-bond-members` with the discovered interface names.
```
